docs/examples/here-data-sdk-aaa-example/.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
Is it not enough to have one .gitignore at the root level?
Why do we need a separate one?

Reply:
updated